dotcom-rendering/src/web/components/ArticlePage.tsx

```diff
@@ -49,50 +57,65 @@ export const ArticlePage = ({ article, NAV, format }: Props) => {
 				`}
 			/>
 			<SkipTo id="maincontent" label="Skip to main content" />
-			<SkipTo id="navigation" label="Skip to navigation" />
-			{(format.design === ArticleDesign.LiveBlog ||
-				format.design === ArticleDesign.DeadBlog) && (
-				<SkipTo id={'key-events-carousel'} label="Skip to key events" />
-			)}
-			<Island clientOnly={true} deferUntil="idle">
-				<AlreadyVisited />
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<FocusStyles />
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<Metrics
-					commercialMetricsEnabled={
-						!!article.config.switches.commercialMetrics
-					}
-				/>
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<BrazeMessaging idApiUrl={article.config.idApiUrl} />
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<ReaderRevenueDev
-					shouldHideReaderRevenue={article.shouldHideReaderRevenue}
-				/>
-			</Island>
-			<Island clientOnly={true} deferUntil="idle">
-				<FetchCommentCounts repeat={true} />
-			</Island>
-			{format.design === ArticleDesign.LiveBlog && (
-				<Island clientOnly={true} deferUntil="idle">
-					<AnimatePulsingDots />
-				</Island>
-			)}
-			<Island clientOnly={true}>
-				<SetABTests
-					abTestSwitches={filterABTestSwitches(
-						article.config.switches,
+			{renderingTarget === 'Web' && (
```

Comment:
On Apps we'll still want to keep the following:
- Main content & key events <SkipTo> links
- FocusStyles
- FetchCommentCounts
- AnimatePulsingDots

It also looks like 'skip to navigation' might have been removed completely? We'll probably want to keep this for web!